static/modes/ada-mode.ts

```diff
@@ -22,9 +22,11 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import * as monaco from 'monaco-editor';
+'use strict';
```

Comment:
Isn't TS strict by default?

Reply:
It should be yeah. This was here previously and I'll remove in the next PR re-applying d660703e757e427b2e8c01ee4b61b32836f9620b and fixing remaining issues